JavaScriptSPA/index.html

```diff
@@ -20,8 +20,8 @@ <h4 id="WelcomeMessage"></h4>
       // Config object to be passed to Msal on creation
       const msalConfig = {
         auth: {
-          clientId: 'Enter_the_Application_Id_here', //This is your client ID
-          authority: "https://login.microsoftonline.com/Enter_the_Tenant_Info_Here", //This is your tenant info
+          clientId: 'Enter_the_Application_Id_Here', //This is your client ID
```

Comment:
You need to update this right? https://github.com/Azure-Samples/active-directory-javascript-graphapi-v2/blob/ecf0427648b638cb9c6cc93deceec2446d7464dc/AppCreationScripts/apps.json

Reply:
I believe "apps.json" is from the older version of script generator. It should be "sample.json" now, with slightly different structure

Reply:
I believe its still apps.json @archieag to confirm

Reply:
apologies, my bad. they are meant for different script generators.

Reply:
@pkanher617 Let me know if you need help updating the apps.json

Reply:
Thank @derisen for updating apps.json

Reply:
no worries! one more thing: "Enter_the_Cloud_Instance_Id_Here/Enter_the_Tenant_Info_Here" -no slash here, should be just "Enter_the_Cloud_Instance_Id_HereEnter_the_Tenant_Info_Here"

Same for "Enter_the_Graph_Endpoint_Here/v1.0/me" -should be just "Enter_the_Graph_Endpoint_Herev1.0/me"

Reply:
Done.